constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/utils/KeyCycleOscillator.java

```diff
@@ -416,7 +256,7 @@ private static void swap(int[] array, float[] value1, float[] value2, int a, int
     }
 
     static class CycleOscillator {
-        static final int UNSET = ConstraintLayout.LayoutParams.UNSET;
+        static final int UNSET = -1;
```

Comment:
Might want to leave a comment explaining this magic number
```suggestion
        static final int UNSET = -1; // ConstraintLayout.LayoutParams.UNSET
```